engine/stats.py

```diff
@@ -31,6 +33,31 @@ def create_stats_dataframe(input_dir, file_id, stats_file_name, file_specificati
     return df
 
 
+def process_member(
+    m_num,
+    member_type,
+    model_output_dir,
+    perturbed_model_output_dir,
+    file_id,
+    stats_file_name,
+    file_specification,
+):
+    if m_num == "ref":
+        input_dir = model_output_dir
```

Comment:
The input_dir name for the reference data has a different format, i.e. it doesn't have the appendix `_member_id_#.` I'm not sure how to handle this, if should otherwise be the same...

Reply:
Of course I could use 0 instead of "ref" but I would still have to use a different input_dir. Or do you have any better idea?

Reply:
Sorry I was confused.